beta-src/src/components/svgr-components/ActionIconSelected.js

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
```

Comment:
ok, i know we talked about this being ignored by eslint before, but since things have changed a little bit since then, i think we should make these TS components now because we want to be sure no one passes an invalid iconState by enforcing a type. this requires undoing the eslintignore change. we need to fix this component up so that is has the same TS usage as our other components.

Reply:
Makes sense! I also updated the svgr command so that it generates typescript commands now, and removed the svgr-components folder from eslintignore.